package.json

```diff
@@ -67,7 +65,8 @@
     "preinstall": "npm prune",
     "postinstall": "bower prune; bower install; electron-rebuild",
     "start": "gulp",
-    "package": "electron-packager . 'Black Screen' --overwrite --platform=darwin --arch=x64 --version='0.31.2' --out='/Applications' --icon='./icon.icns'",
+    "package-windows": "electron-packager . \"Black Screen\" --prune --overwrite --platform=\"win32\" --arch=\"ia32\" --version=\"0.33.0\" --out=\"./.build\" --icon=\"./static/icon.ico\"",
```

Comment:
That's not exactly how I'd like it to work: we should have a single command no matter what platform a person uses.